src/ET_Client.php

```diff
@@ -130,6 +132,11 @@ function __construct($getWSDL = false, $debug = false, $params = null)
 				}
 			}
             if (array_key_exists('accountId', $config)){$this->accountId = $config['accountId'];}
+
+            if (array_key_exists('applicationType', $config)){$this->applicationType = $config['applicationType'];}
```

Comment:
Please change the code in this section (and below where it's the case) to PSR-2 standard:
        if (array_key_exists('applicationType', $config)) {
            $this->applicationType = $config['applicationType'];
        }